incubator/sentry/Chart.yaml

```diff
@@ -0,0 +1,11 @@
+apiVersion: v1
+description: Sentry is a cross-platform crash reporting and aggregation platform.
+name: sentry
+version: 0.1.0
+keywords:
+  - debugging
+  - logging
+sources:
+  - https://github.com/getsentry/sentry
+home: https://sentry.io/
```

Comment:
Add maintainers please use your github name under the name section and an email address